mlflow/system_metrics/metrics/base_metrics_monitor.py

```diff
@@ -0,0 +1,34 @@
+"""Base class of system metrics monitor."""
+import abc
+from collections import defaultdict
+
+
+class BaseMetricsMonitor(abc.ABC):
+    """Base class of system metrics monitor.
+
+    Args:
+        name: string, name of the monitor.
+    """
+
+    def __init__(self, name):
+        self.name = name
+
```

Comment:
```suggestion
```

Reply:
ohh.. it is a separation between public attributes from private

Reply:
Ok, didn't know that convention.